CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [2.0.5] - 2022-03-03
```

Comment:
Based on our experience with osparc-simcore, I think it would be better to release using tags and add changelogs in the github release messages. 

In addition, most of the details you write below are not interesting for the final user therefore there should be a separation based on the audience. Similar will happen with the versions (a draft was proposed in the service integration library)

Reply:
> Based on our experience with osparc-simcore, I think it would be better to release using tags and add changelogs in the github release messages.

We might want to talk about the strategy and how to handle it.

> In addition, most of the details you write below are not interesting for the final user therefore there should be a separation based on the audience. Similar will happen with the versions (a draft was proposed in the service integration library)

I tried to simplify it a bit. This changeling is dedicated to the users and is bundled with the service.

Reply:
i am not convinced but I guess we can tackle that in a different iteration ...